commission/models/commission_settlement.py

```diff
@@ -69,7 +69,7 @@ def _default_company_id(self):
     @api.depends("line_ids", "line_ids.settled_amount")
     def _compute_total(self):
         for record in self:
-            record.total = sum(record.mapped("line_ids.settled_amount"))
+            record.total = sum(record.line_ids.mapped("settled_amount"))
```

Comment:
No need of this diff. It works the same.

Reply:
I think it looks cleaner like this :thinking: Should I not change these then?

Reply:
The principle is to reduce the diff at maximum, specially with cosmetic changes.